audience_manager/api.rst

```diff
@@ -181,33 +223,42 @@ Code::
 
     .. attribute:: user_id
 
-        If application lets user login this option will let Audience Managed better identify user across sessions. It's
-        value should be unique user identifier provided by the App. It may be user handle or internal user ID. Example::
+        If :term:`application` lets :term:`user` to sign in - it's possible to pass unique permanent :term:`user ID`
+        using this option. This will let Audience Manager better identify :term:`user` across devices (laptop, phone)
+        and sessions.
+
+        Example::
 
             "jbond"
 
     .. attribute:: device_id
 
-        If application has access to device id this option will let Audience Managed better identify user across
-        sessions. Example::
+        If :term:`application` has access to :term:`device ID` - it's possible to pass this value using this option.
+        This will let Audience Manager better identify :term:`user` across sessions.
```

Comment:
AM will better identify the user across devices. Sessions are technically correct here, but all of the identifiers allow to identify the user across sessions while device ID gives the ability of cross-device tracking and that should be mentioned here.

Reply:
Is it true? It let's AM identify device. To use it to track user across devices it's necessary to introduce concept of profile merging.

Reply:
You're right, device ID itself doesn't give the ability of cross-device tracking. To achieve that, profiles have to be merged and we need the same identifier sent from different devices (i.e. user ID or email).